cmd/operator/README.md

```diff
@@ -60,10 +60,17 @@ spec:
     - name: https_proxy
       value: http://proxy.host:8433
     - name: no_proxy
-      value: 127.0.0.1,localhost,.example.com,10.0.0.0/8,.svc
+      value: 127.0.0.1,localhost,.example.com,10.0.0.0/8,.svc,.svc.cluster.local
     ...
 ```
 
+**Note:** To build clusters using `kubeadm` with the right `no_proxy` settings from the very beginning,
+set the cluster service names to `$no_proxy` before `kubeadm init`:
+
+```
+$ export no_proxy=$no_proxy,.svc,.svc.cluster.local
```

Comment:
That's a very good tip! Could you please add it also to https://github.com/intel/intel-device-plugins-for-kubernetes/tree/master/cmd/fpga_admissionwebhook#pre-requisites?

Reply:
done!